src/teams.rs

```diff
@@ -134,44 +141,53 @@ impl OrgTeamActions {
     }
 
     /// list the team
-    pub fn get(&self) -> Future<Team> {
-        self.github.get(&self.path(""))
+    pub async fn get(&self) -> Result<Team> {
+        self.github.get(&self.path("")).await
     }
 
     /// edit the team
-    pub fn update(&self, team_options: &TeamOptions) -> Future<Team> {
-        self.github.patch(&self.path(""), json!(team_options))
+    pub async fn update(&self, team_options: &TeamOptions) -> Result<Team> {
+        self.github
+            .patch(&self.path(""), json!(team_options)?)
+            .await
     }
 
     /// delete the team
-    pub fn delete(&self) -> Future<()> {
-        self.github.delete(&self.path(""))
+    pub async fn delete(&self) -> Result<()> {
+        self.github.delete(&self.path("")).await
     }
 
     /// list of teams for this org
-    pub fn list_members(&self) -> Future<Vec<User>> {
-        self.github.get(&self.path("/members"))
+    pub async fn list_members(&self, role: Option<TeamMemberRole>) -> Result<Vec<User>> {
+        self.github.get(self.path("/members")).await
```

Comment:
```suggestion
        self.github.get(&self.path("/members")).await
```

Reply:
Ah man, this wasn't supposed to be part of this diff.